test/system/130-kill.bats

```diff
@@ -153,4 +155,18 @@ load helpers
     run_podman wait $random_name
 }
 
+@test "podman kill - no restart" {
+    ctr=$(random_string 10)
+    run_podman run -d --restart=always --name=$ctr $IMAGE \
+        sh -c "trap 'exit 42' SIGTERM; echo READY; while :; do sleep 0.2; done"
+    run_podman container inspect  --format "{{.State.Status}}" $ctr
+    is "$output" "running" "make sure container is running"
```

Comment:
Suggestion:
```diff
     is "$output" "running" "make sure container is running"
+    run_podman container inspect  --format "{{.State.StoppedByUser}}" $ctr
+    assert "$output" = "false" ".State.StoppedByUser, before kill"
     # Send SIGTERM and make sure the container exits.
```
(And changing line 169 to add "after kill").

I recommend using `assert` over `is`, because the error messages on failure are better.

Reply:
Thanks!  Will update if I need to repush.  If not, I'll remember to use `assert` in the future.